tensorflow/contrib/learn/python/learn/graph_actions.py

```diff
@@ -285,8 +285,8 @@ def make_saver():
         hooks=all_hooks) as super_sess:
       loss = None
       while not super_sess.should_stop():
-        _, loss = super_sess.run([train_op, loss_op], feed_fn() if feed_fn else
-                                 None)
+        _, loss = super_sess.run([train_op, loss_op], feed_fn() if feed_fn else None)
+        
```

Comment:
Can you revert this completely? I think the break was there for the 80 character limit.